### PR TITLE
Translation keys

### LINKED ISSUE #22: 任务文本/物品使用本地化key
可以通过加载本地化文件加载翻译，就可以不用覆盖任务等文件从而达到任务本地化，任务本地化详细方法见:https://www.mcmod.cn/post/2194.html

### PR DESCRIPTION
**Pull request by JieningYu**
this will close https://github.com/DM-Earth/Cabricality/issues/22

TODO:

- [x] kubeJS content
- [x] ponder
- [x] FTB quests

### CODE REVIEW

**Comment by JieningYu** [comment]
this is sooooooooooooooooooooo boring

**Comment by IcedDog** [comment]
oh, as a side note, the ponder language file seems broken
i cant get it to use the translation file and i have no idea why

File: config/ftbquests/quests/chapters/stage_2.snbt [hunk 1]
```diff
@@ -461,15 +424,11 @@
 			}]
 		}
 		{
-			title: "Chapter 2 Complete!"
+			title: ""
```
Comment (JieningYu) [inline config/ftbquests/quests/chapters/stage_2.snbt]: why it's empty here

**Comment by CatastropheChou** [comment]
真是浩大的工程
加油

File: config/ftbquests/quests/chapters/stage_2.snbt (hunk 1 above)
Comment (KrLite) [reply config/ftbquests/quests/chapters/stage_2.snbt]: May be eaten by aliens.

**Comment by JieningYu** [comment]
such huge architecture

**Comment by IcedDog** [comment]
👍

**Comment by IcedDog** [comment]
> oh, as a side note, the ponder language file seems broken i cant get it to use the translation file and i have no idea why

this is fixed after i reinstall my system 
im lost

**Comment by JieningYu** [comment]
@IcedDog so..e what caused that

**Comment by IcedDog** [comment]
> @IcedDog so..e what caused that

idk, do you guys have this problem?

**Comment by JieningYu** [comment]
I don't

**Comment by JieningYu** [comment]
ew there's another *tiny* problem that rhino2 has stricter grammar so the JS codes need to be opti.

**Comment by JieningYu** [comment]
Bruh all the works can be done effortlessly by [this](https://github.com/JieningYu/ftb-quest-lang-generator).

**Comment by IcedDog** [comment]
bruh

**Pull request merged**